frontend_tests/puppeteer_tests/message-basics.ts

```diff
@@ -256,21 +256,6 @@ async function search_tests(page: Page): Promise<void> {
     );
 }
 
-async function expect_all_pm(page: Page): Promise<void> {
-    await page.waitForSelector("#zfilt", {visible: true});
-    await common.check_messages_sent(page, "zfilt", [
-        ["You and Cordelia, Lear's daughter, King Hamlet", ["group pm a", "group pm b"]],
-        ["You and Cordelia, Lear's daughter", ["pm c"]],
-        ["You and Cordelia, Lear's daughter, King Hamlet", ["group pm d"]],
-        ["You and Cordelia, Lear's daughter", ["pm e"]],
-    ]);
-    assert.strictEqual(
-        await common.get_text_from_selector(page, "#left_bar_compose_stream_button_big"),
-        "New stream message",
-    );
-    assert.strictEqual(await page.title(), "Private messages - Zulip Dev - Zulip");
-}
-
```

Comment:
I have removed the checks in puppeteer tests that tested narrowing to all private message aka `#narrow/is/private` as following changes in this commit we have no element in the screen that navigates to this narrow.
I'll add these checks again in the commit in which I add a way for navigating to all private message view(design for which is under discussion).